modules/core/src/persistence.cpp

```diff
@@ -4025,7 +4025,14 @@ static void
 icvJSONWriteReal( CvFileStorage* fs, const char* key, double value )
 {
     char buf[128];
-    icvJSONWrite( fs, key, icvDoubleToString( buf, value ));
+    int len = (int)strlen( icvDoubleToString( buf, value ) );
```

Comment:
I'm not sure that we need to force `size_t` here.
I see here potential problem about `buf` array size (128) that doesn't used/passed anywhere. But this problem is not related to this patch.